src/components/SearchResultTags.js

```diff
@@ -39,6 +40,10 @@ type SearchResultTagsProps = {
    * Defaults to the value in the configuration.
    */
   baseUri?: string,
+  /** Whether to show a Comment option. Defaults to false. */
+  comments: boolean;
+  /** Table field for the comment documents */
+  commentsTable: string | null;
```

Comment:
Indicate commentsTable is an optional prop in the flow annotation. Note, that passing commentsTable as null is not the same as not providing commentsTable prop at all.
```suggestion
  commentsTable: string | null;
```